src/config.ts

```diff
@@ -206,7 +206,7 @@ const defaultConfig: IConfig = {
         username: "name",
         password: "pass",
     },
-    mas: {
+    MAS: {
```

Comment:
the sample config.yaml will also need updating

Reply:
config is updated, can you just double-check the comment I added is clear/sufficient?

Reply:
looks great!